docs/bot/configuration.md

```diff
@@ -43,3 +43,43 @@ These are the keys you can specify:
   "contributors": []
 }
 ```
+
+#### Moving The All Contributors Table Into Different File
+
+As you know, the default file for the contributors table is `README.md`. But, if your project have tons of contributors, it doesn't make sense to includes the All Contributors Table in the `README.md` file. Instead, you can use a different file to do so.
+
+Here is the steps that may help you in achieving that. For the sake of this tutorial, the different file that we're going to be using is going to be named as `CONTRIBUTORS.md`.
+
+1. Install the bot. Read the [Installation]() page.
+2. Add your first contributor. Read the [Bot Usage]() page.
+3. Create the `CONTRIBUTORS.md` file.
+4. Copy the code from the Contributors section in the `README.md` file. Starting from the `ALL-CONTRIBUTORS-LIST:START` line and ending with `ALL-CONTRIBUTORS-LIST:END` line.
+
+Example:
+```
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+<table>
+  <tr>
+    <td align="center">...</td>
+  </tr>
+</table>
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->
+```
```

Comment:
This entire block should be indented so it will appear as part of the item 4.

Reply:
Got it.